apply-roles

```diff
@@ -64,6 +73,21 @@ create_custom_role() {
   echo " - Custom role definition created (${custom_role_definition_id})"
 }
 
+update_custom_role() {
```

Comment:
❓ Wondering where in the code we are calling this function?

Reply:
Yeah bash and the PR diff don't make it very obvious but it is called on [line 196](https://github.com/cloudconformity/azure-onboarding-scripts/pull/18/files#diff-7a1895416f0c412b1b211e7300d7c395b09f62d30d81097b9cebbeeabaa57752R196) when the `--update-role` is passed

Reply:
Sweet! 
I must have totally missed that.